src/MizarData/include/MizarData/SamplingWithFrameTrackComparisonReport.h

```diff
@@ -12,11 +12,22 @@
 #include <utility>
 
 #include "ClientData/ScopeStats.h"
+#include "MizarData/BaselineOrComparison.h"
 #include "MizarData/SampledFunctionId.h"
 #include "OrbitBase/Logging.h"
 
 namespace orbit_mizar_data {
 
+// Whatever is usually referred to as "Statistical Test" we call "Comparison" in the project to save
+// confusion with Unit tests.
+struct ComparisonResult {
+  double statistic{};
+
+  // The term from Statistics. TL;DR: The smaller it is, the less we believe in the assumption under
+  // test (e.g. no difference in active function time).
+  double pvalue{};
```

Comment:
`p_value` ?

Reply:
I am not sure. Actually, it is "p-value", not "p value", hence I am reluctant to use `p_value`. But I've got a feeling, like in the core it's `pvalue` more often. OTOH, I've never seen how p-values are referred to in C++.